std/algorithm/sorting.d

```diff
@@ -2889,18 +2889,46 @@ void partialSort(alias less = "a < b", SwapStrategy ss = SwapStrategy.unstable,
     Range)(Range r, size_t n)
     if (isRandomAccessRange!(Range) && hasLength!(Range) && hasSlicing!(Range))
 {
-    topN!(less, ss)(r, n);
-    sort!(less, ss)(r[0 .. n]);
+    partialSort!(less, ss)(r[0 .. n], r[n .. $]);
 }
 
 ///
-@safe unittest
```

Comment:
These annotations are there on purpose, e.g. to ensure that we don't break user code!
Instead of removing annotations them we should rather add them ...